integrations/aws/utils/resources.py

```diff
@@ -209,6 +226,16 @@ async def resync_cloudcontrol(
                     ]
 
                 for instance in resources:
+                    if isinstance(instance, Exception):
+                        if is_resource_not_found_exception(instance):
+                            error = typing.cast(ClientError, instance)
+                            logger.warning(
+                                f"Skipping resyncing {kind} resource in region {region} in account {account_id}; {error.response['Error']['Message']}"
+                            )
```

Comment:
how the error will look like? if we know that its resource not found, can't we provide more accurate log? I also think that its a info rather than a warning

Reply:
`Skipping resyncing AWS::EC2::Instance resource in region eu-north-1 in account 213138413742; Resource of type 'AWS::EC2::Instance' with identifier 'i-0dfe65a174b846790' was not found`